### DISCUSSION TITLE
Purpose of chromium as a Quarto tool?

### DISCUSSION THREAD

**Discussion by dmbates** [quarto-dev/quarto-cli#2424] [Q&A]
`quarto tools list` shows that chromium is available as a optional tool for Quarto.  I use Linux (Ubuntu 22.04) with firefox installed.  When I preview a quarto document (with Julia code chunks executed through Jupyter) I sometimes get warnings like
```
Gtk-Message: 09:00:50.757: Failed to load module "canberra-gtk-module"
Gtk-Message: 09:00:50.758: Failed to load module "canberra-gtk-module"
ATTENTION: default value of option mesa_glthread overridden by environment.
ATTENTION: default value of option mesa_glthread overridden by environment.
ATTENTION: default value of option mesa_glthread overridden by environment.
Missing chrome or resource URL: resource://gre/modules/UpdateListener.sys.mjs
```
I tried installing chromium through

```
quarto tools install chromium
```

and do get an installation in `~/.local/share/quarto/chromium/` but this does not seem to affect the preview.

P.S. Are there Ubuntu packages I should add to address the canberra module loading warnings?

**Comment by cderv** [comment]
Chromium is among the tools offer for install as it is required to Diagrams feature in PDF and DOCX outputs (where diagrams are printed to images)

See https://quarto.org/docs/authoring/diagrams.html

Thanks for the report of the issue above. It is rather cryptic. Can you share a document where this happens so we can reproduce ? 
Are you using Diagrams ?
